File: src/Hoops.Core/Models/Location.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Hoops.Core.Interface;

namespace Hoops.Core.Models
{
    [Table("ScheduleLocations")]
    public partial class Location : IAuditable
    {
        [Key]
        public int LocationNumber { get; set; }
        public string LocationName { get; set; }
        public string Notes { get; set; }

        public DateTime? CreatedDate { get; set; }
        public int? CreatedUser { get; set; }
        public DateTime? ModifiedDate { get; set; }
        public int? ModifiedUser { get; set; }
    }
}

File: hoops.ui/src/domain/location.ts
export class Location {
  public locationNumber!: number;
  public locationName!: string;
  public notes!: string;
  public createdDate?: Date | string | null;
  public createdUser?: number | null;
  public modifiedDate?: Date | string | null;
  public modifiedUser?: number | null;
}
